prometheus_client/samples.py

```diff
@@ -4,30 +4,30 @@
 class Timestamp:
     """A nanosecond-resolution timestamp."""
 
-    def __init__(self, sec, nsec):
+    def __init__(self, sec: Union[int, float], nsec: Union[int, float]) -> None:
         if nsec < 0 or nsec >= 1e9:
             raise ValueError(f"Invalid value for nanoseconds in Timestamp: {nsec}")
         if sec < 0:
             nsec = -nsec
-        self.sec = int(sec)
-        self.nsec = int(nsec)
+        self.sec: int = int(sec)
```

Comment:
did mypy complain here when theres an explicit int() cast but it didnt infer the type fro that?

Reply:
Ahh, good catch. I think this was from a test with pyright which somehow didn't infer it from the cast. We aren't using pyright yet so I removed them.

Reply:
It is a good idea to describe the types of public fields in the body of the class. Other users can use pyright for type checking their projects and pyright doesn't trying to infer types in such cases (`__init__` can be much more complicated), in py.typed project all public members should be type annotated.

Reply:
Interesting, thanks for the link! I updated this code back to having explicit int types and looked through the other classes assigning to public variables in this PR and it looks like they are covered. Hopefully that helps out pyright users while we work on getting more coverage.